src/index.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { createRoot } from 'react-dom/client';
 import App from './ui/App';
+import './i18n/i18n';
```

Comment:
After `src/i18n/i18n.ts` is renamed to `src/i18n/index.ts` it should be just `import './i18n'`